model/src/main/java/com/javahelp/model/token/Token.java

```diff
@@ -0,0 +1,79 @@
+package com.javahelp.model.token;
+
+import com.javahelp.model.user.User;
+
+import java.time.*;
+import java.security.SecureRandom;
```

Comment:
Might wanna clean up these imports but tbh i don't think it's a huge deal we can also do a general imports, cleanup, and formatting PR in the future.

Reply:
If you're changing other stuff you might as well do these too though, you can just hit Ctrl + Alt + O to automatically clean them.